testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/ClientlibsIncludeIT.java

```diff
@@ -0,0 +1,73 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2021 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.cq.wcm.core.components.it.http;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.testing.clients.ClientException;
+import org.junit.*;
+import org.junit.rules.ErrorCollector;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+
+public class ClientlibsIncludeIT {
+
+    @ClassRule
+    public static final CQAuthorPublishClassRule cqBaseClassRule = new CQAuthorPublishClassRule();
+
+    @Rule
+    public CQRule cqBaseRule = new CQRule(cqBaseClassRule.authorRule, cqBaseClassRule.publishRule);
+
+    @Rule
+    public ErrorCollector collector = new ErrorCollector();
+
+    private static CQClient adminAuthor;
+    private String testPage = "/content/core-components/clientlibs-include-page";
+    private final static String CONTAINER_SCRIPT_ELEMENT = "<script async crossorigin=\"anonymous\" onload=\"console.log()\" src=\"/etc.clientlibs/core/wcm/components/commons/site/clientlibs/container.lc-1197d358a0a463b3e0891f4ed50e4864-lc.min.js\"></script>";
```

Comment:
Aren't the cache killer `lc-1197d358a0a463b3e0891f4ed50e4864-lc` going to change based on when these specific clientlibs get installed/updated?

Reply:
addressed in https://github.com/adobe/aem-core-wcm-components/pull/1881/commits/73b4f90e6876db87c77df226df80defe0d870e5f

Reply:
LGTM now